src/__tests__/downshift.get-button-props.js

```diff
@@ -19,10 +19,12 @@ test('space on button opens and closes the menu', () => {
 test('clicking on the button opens and closes the menu', () => {
   const {button, renderSpy} = setup()
   Simulate.click(button)
+  jest.runAllTimers()
```

Comment:
Turns out this is required by pretty much anyone testing a component that uses downshift and it's a bit annoying. @aMollusk, could you jump into https://github.com/paypal/downshift/issues/437 and chime in to discuss an alternative/workaround for tests?